### PR TITLE
homework5

### PR DESCRIPTION
**Pull request by zadziarnouksi-taras**
For checking!

### CODE REVIEW

File: homework5/numberoffate.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * numberoffate.cpp: NUMBER OF FATE. The digits of the full date of birth are added together until one digit is obtained.
+ * Design a function and calculate your fate number.
+ * */
+
+#define MIN_DAY 1
+#define MAX_DAY 31
+#define MIN_MONTH 1
+#define MAX_MONTH 12
+#define MIN_YEAR 1960
+#define MAX_YEAR 2015
```
Comment (VadzimAlkhimionak) [inline homework5/numberoffate.cpp:15]: Cool!

File: homework5/numberoffate.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+int calcNumberOfFate(int day, int month, int year) {
+    return calcSumOfDigits(day) + calcSumOfDigits(month) + calcSumOfDigits(year);
+}
+
+int calcSumOfDigits(int number) {
+    int sum = 0;
+
+    if (number / 10 == 0) return number;
```
Comment (VadzimAlkhimionak) [inline homework5/numberoffate.cpp:67]: `number / 10 == 0` or `number / 10 <= 0` ?

**Review by VadzimAlkhimionak — COMMENTED** [review]
Good job!

File: homework5/numberoffate.cpp (hunk 1 above)
Comment (zadziarnouksi-taras) [reply homework5/numberoffate.cpp:15]: Thank you! :)

File: homework5/numberoffate.cpp (hunk 2 above)
Comment (zadziarnouksi-taras) [reply homework5/numberoffate.cpp:67]: `number / 10 == 0` this option, because if `number = 7` then `7 / 10 == 0 ` - true and return `7`.

**Pull request merged**